scheduler/models.py

```diff
@@ -20,6 +20,11 @@ class CourseTable(models.Model):
     time = models.CharField(max_length=30, blank=True, null=True)
 
 
+class CourseTA(models.Model):
```

Comment:
Why arent you using the join table? it accomplishes the same as this, right?

Reply:
I thought thats what I was doing